docs/monitors/df.md

```diff
@@ -0,0 +1,79 @@
+<!--- GENERATED BY gomplate from scripts/docs/monitor-page.md.tmpl --->
+
+# df
```

Comment:
This should probably be called `filesystems` or something more descriptive than `df`.